modules/custom/social_language/modules/social_content_translation/social_content_translation.install

```diff
@@ -78,14 +78,15 @@ function social_content_translation_disable_field_translations() {
  * Add support translations for entities.
  */
 function social_content_translation_update_8001() {
-  $module_names = [
-    'social_group_flexible_group',
-  ];
-
-  $config = \Drupal::configFactory()->getEditable('social_content_translation.settings');
+  // Don't do anything.
+}
 
-  foreach ($module_names as $module_name) {
-    $config->set($module_name, TRUE);
-    $config->save(TRUE);
-  }
+/**
+ * Delete redundant configuration settings.
+ */
+function social_content_translation_update_8002() {
```

Comment:
@nechai As we agreed better to start hook update with the target OS version, so it will be 10301

Reply:
resolved